data/changelog/ppxlib/2024-07-25-ppxlib-0.33.0.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Ppxlib 0.33.0
+tags: [ppxlib, platform]
+changelog: |
+  - Fix a bug where `Code_path.main_module_name` would not properly remove
+  extensions from the filename and therefore return an invalid module name.
+  (#512, @NathanReb)
+
+  - Add `-unused-type-warnings` flag to the driver to allow users to disable
+  only the generation of warning 34 silencing structure items when using
+  `[@@deriving ...]` on type declarations. (#511, @mbarbin, @NathanReb)
+
+  - Make `-unused-code-warnings` flag to the driver also controls the generation
+  of warning 34 silencing structure items when using `[@@deriving ...]` on type
+  declarations. (#510, @mbarbin, @NathanReb)
+
+  - Driver: Add `-unused-code-warnings=force` command-line flag argument. (#490, @mbarbin)
+
+  - New functions `Ast_builder.{e,p}list_tail` that take an extra tail
```

Comment:
Does there need to be a verb here, like "Add new functions" or "Create new functions"?
All other items in this list start with a verb